### PR TITLE
Port #51993 to master

### PR DESCRIPTION
**Pull request by alexey-zhukovin**
### What issues does this PR fix or reference?
Port #51993 to master

### Commits signed with GPG?
Yes/No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

File: salt/modules/flatpak.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+"""
+Manage flatpak packages via Salt
+
+.. versionadded:: Neon
```
Comment (s0undt3ch) [inline salt/modules/flatpak.py:5]: Please change Neon to Sodium

File: salt/states/flatpak.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+"""
+Management of flatpak packages
+==============================
+Allows the installation and uninstallation of flatpak packages.
+
+.. versionadded:: Neon
```
Comment (s0undt3ch) [inline salt/states/flatpak.py:7]: Same here

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

**Comment by waynew** [comment]
I'm working on adding tests for this, I believe I'll have 'em done on Thursday

**Review by waynew — CHANGES_REQUESTED** [review]
needs tests (I've been working through adding them during test clinic)

**Comment by fzoske** [comment]
@waynew Any news on this pullrequest? Would be super happy to see this get merged :)

**Comment by dwoz** [comment]
Closing this due to inactivity. Anyone should feel free to re-open it if they want to see it through to the end in one release cycle.

**Pull request closed without merging**